wgpu-types/src/lib.rs

```diff
@@ -1168,27 +1571,38 @@ impl<L> SamplerDescriptor<L> {
     }
 }
 
+/// Describes a [`CommandBuffer`].
 #[repr(C)]
 #[derive(Clone, Debug, Default, PartialEq, Eq, Hash)]
 #[cfg_attr(feature = "trace", derive(Serialize))]
 #[cfg_attr(feature = "replay", derive(Deserialize))]
 pub struct CommandBufferDescriptor {
+    /// Set this member to zero
     pub todo: u32,
 }
 
+/// Describes a [`RenderBundleEncoder`].
 #[derive(Clone, Debug, Default, PartialEq, Eq, Hash)]
 pub struct RenderBundleEncoderDescriptor<'a> {
+    /// Debug label of the render bundle encoder. This will show up in graphics debuggers for easy identification.
     pub label: Option<&'a str>,
+    /// The formats of the color attachments that this render bundle is capable to rendering to. This
+    /// must match the formats of the color attachments in the renderpass this render bundle is executed in.
     pub color_formats: &'a [TextureFormat],
+    /// The formats of the depth attachment that this render bundle is capable to rendering to. This
+    /// must match the formats of the depth attachments in the renderpass this render bundle is executed in.
     pub depth_stencil_format: Option<TextureFormat>,
+    /// Sample count this render bundle is capable of rendering to. This must match the pipeline
```

Comment:
must match both the pipelines (not a singular pipeline), and the sample count of render passes it's used in